examples/robot_example.cpp

```diff
@@ -60,16 +63,44 @@ class SimpleRobotControlLoop : public kodlab::mjbots::MjbotsControlLoop<ManyMoto
     }
 };
 
+class LimbTest : public kodlab::LimbBase
```

Comment:
Is this class intended to be committed?  Or is this just for your testing?  If the former, this class definition should live in its own file somewhere with proper commenting.  If the latter, be sure to remove before final merge.